formal.rst

```diff
@@ -1,18 +1,53 @@
 .. _formal:
 
 ==========================================
-Normative specification for CellML 2.0
+CellML 2.0 Specification
 ==========================================
 
-The first section here includes the normative specification documentation for the CellML format.
-These represent the official rules about what must and must not be included in order for a model to be syntactically valid CellML.
+This document is the normative version of the CellML Specification, defining the CellML syntax and the rules by which it should be used.
+It is intended primarily for the developers of software tools which directly consume CellML syntax.
+Users of CellML models should read the informative version of the CellML Specification which is available here: **TODO**
 
+**Authors:**
+Michael Clerx,
+Michael T. Cooling,
+Jonathan Cooper,
+Alan Garny,
+Keri Moyle,
+David P. Nickerson,
+Poul Nielsen,
+and Hugh Sorby.
```

Comment:
Shouldn't the "and" be on the previous line? (Same below.)

Reply:
It's all rendered to a single line anyway ...

Reply:
034c2ef done